.github/actions/test-rust/action.yaml

```diff
@@ -15,29 +15,29 @@ runs:
         # Without inputs.target, it seems to select linux x64 in the wasm32
         # tests, though in the docs it says it uses the target as the key?
         key: ${{ inputs.target_option }}
-    - uses: jetli/wasm-bindgen-action@v0.1.0
+    - uses: jetli/wasm-bindgen-action@v0.2.0
       if: inputs.target_option == '--target=wasm32-unknown-unknown'
     - name: 📎 Clippy
-      uses: actions-rs/cargo@v1
+      uses: richb-hanover/cargo@v1.1.0
```

Comment:
Yes, a fork for this and tarpaulin, both in github.com/actions-rs, might be in order. Is this something to consider under the PRQL umbrella? Maybe `prql/actions-rs-cargo `and `prql/actions-rs-tarpaulin`?

I say this because you're way more tuned in to the rust ecosystem and tools. Plus you're firmly committed to keeping this software running.

Reply:
Yeah, I think keeping the prql org focused on the PRQL lang keeps the scope limited, and having a proper home — either some org like `actions-rs` was intending to be, or someone dedicated to maintaining it, will give it the focus it needs.

So if you have the patience, add a comment, but no great stress; I'll try and remember

Reply:
I just wrote a note to the author of that repo, asking if there's someone maintaining them... I'll let you know what I find.

Reply:
PS I'm happy to leave cargo@v1.1.0 there for the time being.